bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/internal/SitemapResource.java

```diff
@@ -235,8 +239,8 @@ public Response getSitemapData(@Context HttpHeaders headers,
             @PathParam("sitemapname") @ApiParam(value = "sitemap name") String sitemapname,
             @QueryParam("type") String type, @QueryParam("jsoncallback") @DefaultValue("callback") String callback) {
         final Locale locale = localeService.getLocale(language);
-        logger.debug("Received HTTP GET request at '{}' for media type '{}'.",
-                new Object[] { uriInfo.getPath(), type });
+        logger.debug("Received HTTP GET request from IP {} at '{}' for media type '{}'.",
+                new Object[] { request.getRemoteAddr(), uriInfo.getPath(), type });
```

Comment:
I know you only added one additional information to the log, but what is the reason to create an object array? Why not using the vararg logging method of slf4j's logger?

Reply:
I was surprised myself but did not want to change something clever that could have been done lol
So I will fix that, this is I think the only one like that in this file.